src/pages/library/DataBrowser.js

```diff
@@ -217,7 +216,7 @@ export const Browser = () => {
 
   const toggleSelectedData = data => setSelectedData(_.xor([data]))
 
-  return h(FooterWrapper, { alwaysShow: true }, [
+  return [
```

Comment:
Currently, this results in a React warning: `Warning: Each child in a list should have a unique "key" prop.`.

This can be resolved by returning a Fragment element here instead of an array.

```js
import { Fragment } from 'react'

...

return h(Fragment, [
  h(SearchAndFilterComponent, {
  ...
])

Reply:
Hi @nawatts! I took this over since Phil is OOO for the rest of the week. I think things should be resolved now, would you mind taking a look?